clearwater-snmpd/usr/share/clearwater/infrastructure/scripts/snmpd

```diff
@@ -33,6 +33,13 @@
 # as those licenses appear in the file LICENSE-OPENSSL.
 #
 
+# Do nothing if snmpd.conf is absent (e.g. mid-install)
+
+if [ ! -e /etc/snmp/snmpd.conf ]
+then
+    exit 0
```

Comment:
Rather than exiting here, I think we should wrap the following code in the (negated) if.  As it is, we don't run the last few commands of the script which create a couple of directories and don't strictly relate to `/etc/snmp/snmpd.conf`.